modules/pel/peltool/peltool.py

```diff
@@ -335,7 +335,18 @@ def listOption(path: str, config: Config, extension: str, rev: bool =False):
             file_list.append(file)  ## create list of file names
         # Only process top level directory
         break
-    file_list.sort(reverse=rev) 
+    file_list.sort(reverse=rev)
+    if all:
+        final_string = "[ \n"
```

Comment:
allPELsData?

General tips: I think you need to improve your naming style to make it readable/understandable for everyone.